### ISSUE TITLE
Visual code 1.4 is crashing

### ISSUE THREAD

**Issue by afym** [Microsoft/vscode#10325]
- VSCode Version: 1.4
- OS Version: Windows 8.0

Steps to Reproduce:

1. Update Visual code to the new 1.4 version
2. Re-open project
3. Visual code is slow and try to load my project (simple php app)
4. Visual code crashes
5. Close visual code and try again (same problem over and over)
6. Uninstall and reinstall but the problem continuous

Can someone help me?

**Comment by bpasero** [comment]
I am very sorry to hear you are having issues with VS Code. I wonder if this issue is being caused by an installed extension. Can you try to run VS Code without extensions? From the command line, execute: `code --disable-extensions` and try your steps again to see if it reproduces. If you see it is an issue with the extension, please file it against the extension repository itself.

**Comment by jrieken** [comment]
@afym Can you try as suggested above to run without extensions? Also can you check with `top` or another task manager which of the VS Code processes is running high?

**Comment by afym** [comment]
Hi there. I used code --disable-extensions but it still crashing. In my task manager there many visual code processes open using 15MB in avarage.

**Comment by jrieken** [comment]
Is there a VS Code process with high CPU load?

**Comment by jayteesanchez** [comment]
Hello, I had the same issue with too many extensions to name but it didn't start happening until after I upgraded to 1.4. I did try uninstalling all of the extensions and running it without any and it still would freeze and then not accept any input from my keyboard. I even tried the VSCODE-insiders version, with and without extensions and ran into the same issue. 

The solution that finally worked for me though was to delete both VSCODE and VSCODE-insiders and reinstall. Hope this helps! 

VSCODE: 1.4 
OS: Mac 10.11.5
